test/unit/coupling/interface/impl/ls1/LS1StaticCommDataTest.cpp

```diff
@@ -0,0 +1,66 @@
+#include <cppunit/TestFixture.h>
+#include <cppunit/extensions/HelperMacros.h>
+
+#include "coupling/interface/impl/ls1/LS1StaticCommData.h"
+
+#include <array>
+
+#if (COUPLING_MD_PARALLEL == COUPLING_MD_YES)
+#include <mpi.h>
+#endif
+
+#define MY_LINKEDCELL ls1::LS1RegionWrapper
+
+class LS1StaticCommDataTest : public CppUnit::TestFixture
+{
+	CPPUNIT_TEST_SUITE(LS1StaticCommDataTest);
+	CPPUNIT_TEST(testSerialFunctions);
+	CPPUNIT_TEST(testParallelFunctions);
+	CPPUNIT_TEST_SUITE_END();
+
+public:
+	void setUp()
+	{
+		
+	}
+	void tearDown()
+	{
+
+	}
+	void testSerialFunctions()
+	{
+		std::string filename = "test";
+		coupling::interface::LS1StaticCommData::getInstance().setConfigFilename(filename);
+		CPPUNIT_ASSERT( coupling::interface::LS1StaticCommData::getInstance().getConfigFilename() == filename );
+
+		std::array<double,3> offsets({5,5,5});
+		for(int i = 0; i < 3; i++)
+			coupling::interface::LS1StaticCommData::getInstance().setBoxOffsetAtDim(i,offsets[i]);
+		for(int i = 0; i < 3; i++)
+			CPPUNIT_ASSERT( coupling::interface::LS1StaticCommData::getInstance().getBoxOffsetAtDim(i) == offsets[i] );
+	}
+	void testParallelFunctions()
```

Comment:
Wouldn't it be better to have this `#if (COUPLING_MD_PARALLEL == COUPLING_MD_YES)` line 41 and add the test line 18 only if compiling in parallel ? Otherwise we would have an empty test when not parallel that would succeed, could be confusing.

Reply:
You're right, I'll do it